monitor.py

```diff
@@ -46,6 +46,40 @@ def fetch_release_date(release):
         return "Error while sending request to url"
     except json.JSONDecodeError as e:
         return "Failed to extract the spy-links"
+    
+def fetch_build_time(build):
+    '''
+    Returns the created time (HH:MM) and date (YYYY-MM-DD) of the release in IST
+    '''
+    url = constants.PROW_VIEW_URL + build[8:] + '/prowjob.json'
+    
+    try:
+        response = requests.get(url, verify=False, timeout=15)
+        if response.status_code == 200:
+            buildtime = json.loads(response.text)
+            timestamp_str = buildtime["metadata"]["creationTimestamp"]
+        
+        # Convert timestamp to datetime object (UTC)
+            utc_time = datetime.strptime(timestamp_str, "%Y-%m-%dT%H:%M:%SZ")
+        
+        # Convert UTC to IST (UTC +5:30)
+            ist_time = utc_time + timedelta(hours=5, minutes=30)
+        
+        # Extract date and time separately
+            ist_date = ist_time.strftime("%Y-%m-%d")  # YYYY-MM-DD
+            ist_time_formatted = ist_time.strftime("%H:%M")  # HH:MM (without seconds)
+
+            return ist_date, ist_time_formatted  # Returning both date and time
+        else:
+            return 'ERROR'
```

Comment:
This return 'ERROR'  is not handled in the calling funtion. 

In these [line1256 ](https://github.com/ocp-power-automation/ci-monitoring-automation/blob/45d2eb90752964dd7ec5fb754b5c3a56face5f1b/monitor.py#L1256) and [line1323]( https://github.com/ocp-power-automation/ci-monitoring-automation/blob/45d2eb90752964dd7ec5fb754b5c3a56face5f1b/monitor.py#L1323), we are expecting 2 return values but we get only one value which will lead to error during the runtime

Can we handle this in the calling functions.

And Can we ellaborate the error with proper error statement like below
`return f"ERROR: Received unexpected status code {response.status_code}"`